docs/cloning.md

```diff
@@ -12,17 +12,18 @@
 
   **Note** The ansible playbook run will fail if the free space in root partition is less than the value specified in `required_root_free_space` variable in [roles/sat6repro/vars/main.yml] (roles/sat6repro/vars/main.yml)
 
+2. Place the backup files on the Destination node and update the `backup_dir` variable in `vars/main.yml` accordingly.
```

Comment:
updating this variable happens on the control node, but is listed under destination node steps, can we specify that it happens on the control node?